### PR TITLE
Python logging

### LINKED ISSUE #50: python warnings
Use standard module in all python code https://docs.python.org/3/library/warnings.html

### PR DESCRIPTION
**Pull request by fredRos**
Once complete, this fixes #50

### CODE REVIEW

**Comment by fredRos** [comment]
@dvandyk can you post the code we used to test the python plot client? I want to see if exceptions and log messages are created as desired. So far this PR is untested, so don't merge!

**Comment by dvandyk** [comment]
@fredRos ```make -C manual/examples``` should do the trick. Plotting the results is the last step in the examples (for the time being).

**Comment by fredRos** [comment]
Thanks. It's good to know that these examples are run on travis, too. Ready from my side. I just didn't force the log to stderr because that's not common in python.

File: .travis/build-and-test.bash [hunk 1]
```diff
@@ -10,11 +10,9 @@ function build_and_test_ubuntu() {
...
     pushd /build
     export CXXFLAGS="-O2 -g"
-    /src/configure \
-        --enable-pmc \
-        --enable-python \
-        --prefix=/usr
-    make distcheck -j2 DISTCHECK_CONFIGURE_FLAGS="--enable-pmc --enable-python --prefix=/usr" VERBOSE=1
+    CONFIGURE_FLAGS="--enable-pmc --enable-python --prefix=/usr"
+    /src/configure ${CONFIGURE_FLAGS}
+    make distcheck -j2 DISTCHECK_CONFIGURE_FLAGS=${CONFIGURE_FLAGS} VERBOSE=1
```
Comment (dvandyk) [inline .travis/build-and-test.bash]: In bash, ```foo=bar baz``` works to assign a variable. However, when invoking a command white space remain white spaces. You need to wrap ```${CONFIGURE_FLAGS}``` in quotation marks in the ```make distcheck``` line.

**Comment by fredRos** [comment]
Now you have icing

**Comment by fredRos** [comment]
builds pass

**Pull request merged**